WordPress/wordpress-playground
# Remove old PR preview HTML files and add redirects to new preview modals

## Motivation for the change, related issues

This PR removes old PR preview pages, adds relevant redirects, and updates old PR preview redirects.

Fixes #1969

## Implementation details

In addition to the removals and redirects, this PR:
- Removes some more complicated redirect-related logic because it is awkward and no longer needed.
- Adjust old UI links to wordpress.html to link to the new modal instead

## Testing Instructions (or ideally a Blueprint)

- Tested manually on a staging site.
- Use `php -l` to ensure there are no syntax errors in custom-redirects-lib.php
- Deploy and confirm the following paths redirect to the new PR preview modals
  - /wordpress
  - /wordpress.html
  - /gutenberg
  - /gutenberg.html
